src/SDKs/ServiceBus/data-plane/tests/Microsoft.Azure.ServiceBus.Tests/MessageTests.cs

```diff
@@ -140,9 +140,9 @@ public async Task MessageWithMaxMessageSizeShouldWorkAsExpected()
                     var queueClient = new QueueClient(TestUtility.NamespaceConnectionString, queueName, ReceiveMode.PeekLock);
 
                     try
-                    {
-                        var maxMessageSize = (256 * 1024) - 58;     // 58 bytes is the default serialization hit.
-                        var maxPayload = Encoding.ASCII.GetBytes(new string('a', maxMessageSize));
+                    { 
+                        var maxMessageSize = (256 * 1024) - 77;     // 77 bytes is the current serialization hit.
```

Comment:
Unsure why, but I was seeing fairly consistent failures here after some of the asynchronous changes.   I validated the sizes a few different times and a few different ways, and I was consistently seeing the serialization size as `77` and failing with the original code.   The reported max size of the AMQP link was reported consistently in the failure messaging, so I don't believe that there was a buffering or saturation issue.    If anyone has a thought on why this may be, I'd be grateful for the insight.